R/.gitignore

```diff
@@ -0,0 +1,3 @@
+GWAS_Tutorial_Manhattan.png
+globals.R
```

Comment:
Yeah, that's an annoyance.  But I'm OK with us just oscillating back and forth with the data.dir value.
So can you remove that line in the file, commit and push your branch again?  Typically these Pull Requests include some discussion that results in modifications of a branch.  Or to put it differently, a PR is not attached to a specific commit, but is associated with a branch and there can be changes on a branch even after the PR is created up until the PR is accepted and the branch is merged to master.